src/styles/stars.scss

```diff
@@ -0,0 +1,38 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    height: 16px;
+    width: 16px;
+  }
+
+  .stars__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &--5 .stars__star:nth-last-child(n + 2) {
+    background-image: url(../images/star-active.svg);
+  }
+}
+
+.starsReviews {
+  flex-direction: row;
+  margin-top: 16px;
+
+  width: 166px;
+  justify-content: space-between;
+
+  display: flex;
+  align-items: flex-end;
+}
+
+.reviews {
```

Comment:
The `reviews` class should be part of the `stars` block to comply with BEM naming conventions. Consider renaming it to something like `stars__reviews-text`.